addons/common/functions/fnc_moveCursor.sqf

```diff
@@ -0,0 +1,30 @@
+/*
+ * Author: LorenLuke (function), BaerMitUmlaut (Extension)
+ * Simulates mouse input by number of pixels (not consistent/smooth)
+ * Note- using with open dialogs may cause issues when in loops.
+ *
+ * Arguments:
+ * 0: x direction (+right, -left) <NUMBER>
+ * 1: y direction (+down, -up) <NUMBER>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [100,100] call ace_common_fnc_moveCursor
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_x", "_y"];
+
+if (dialog) exitWith {};
+
+// "bluefcs" callExtension "x100"; "bluefcs" callExtension "y-100"
+// Yes, they're backwards. No, I can't fix it because I don't know how to DLL. -Luke
+call compile format ['"ace_cursor_movement" callExtension "y%1"',_x];
```

Comment:
call compile format? really?
`"ace_cursor_movement" callExtension format ["%1:%2",_x,_y]`

Reply:
~April, April.~ Yes, really.
I do stuff the way I know how that it works. If there's a better way, tell instead of condescending for not knowing the better way.

Reply:
> If there's a better way, tell instead of condescending for not knowing the better way.

Uhm...
>`"ace_cursor_movement" callExtension format ["%1:%2",_x,_y]`

And my other comment about the format in the extension?

Reply:
_*I didn't write the extension. I don't know how to .DLL.*_

Reply:
And? Does that mean problems in your PR don't need to be fixed and can just be ignored?